lib/methods.rb

```diff
@@ -0,0 +1,11 @@
+def parse_product_array(string)#try regex /:|;/
```

Comment:
This can live inside the class that calls it as a helper method! If it won't be used outside the class, don't bother making accessible to others!